packages/renderer/src/lib/feedback/feedbackForms/GitHubIssueFeedback.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+import { Button, ErrorMessage, Link } from '@podman-desktop/ui-svelte';
+
+import FeedbackForm from '../FeedbackForm.svelte';
+
+let issueTitle = '';
+let issueDescription = '';
+let issueValidaionError = '';
+
+$: if (!issueTitle) {
+  if (!issueDescription) {
+    issueValidaionError = 'Please enter bug title and description';
+  } else {
+    issueValidaionError = 'Please enter bug title';
+  }
+} else {
+  if (!issueDescription) {
+    issueValidaionError = 'Please enter bug description';
+  }
+}
+
+export let hideModal = () => {};
```

Comment:
issue: it looks strange to have a method named hideModal that do nothing

either it's a hideModal that do something or if we don't know what it's doing, it's a callback like: beforeOpen or afterOpen, etc and the hide thing is not know by this component

Reply:
this was meant to be the `hideModel` method that is passed from `SendFeedback.svelte`, so that this component could call it when needed.